intrastat_product/models/intrastat_product_declaration.py

```diff
@@ -477,8 +477,8 @@ def _prepare_invoice_domain(self):
         start_date = date(int(self.year), int(self.month), 1)
         end_date = start_date + relativedelta(day=1, months=+1, days=-1)
         domain = [
-            ("invoice_date", ">=", start_date),
-            ("invoice_date", "<=", end_date),
+            ("date", ">=", start_date),
```

Comment:
Are you sure these domains should be by accounting date? One thing is the amount, and the other is in which period to include the invoices.